### DISCUSSION TITLE
User Mode Flatpaks in Software Manager

### DISCUSSION THREAD

**Discussion by xobody** [linuxmint/discussions#517] [Ideas]
Could Software/Update Manager be extended to use flatpak in --user mode instead of (or together with) --system mode? Right now, it only works in --system mode, so it can't detect the presence of --user installed flatpaks to manage them and Update Manager can't update them. System flatpaks have sometimes been awkward to use and update (flood of authorization prompts), so I only use --user mode.

If the --system flathub source is removed from flatpak, Software Manager doesn't see any flatpaks even if a --user flathub source is present.

Software Manager
1. show packages from --user flathub source
2. detect/install/remove --user flatpaks

Update Manager
1. update --user flatpaks

Thanks for reading!

**Comment by ptrj** [comment]
Please add this feature, it seems more people want this feature including me :-)
https://forums.linuxmint.com/viewtopic.php?t=422172
Thanks

**Comment by pflovv** [comment]
What a great idea for both security and convenience! This feature would allow children to explore and install various apps and games within their standard user accounts without needing to know the root or admin passwords. This prevents accidental system changes and maintains overall system security while providing flexibility and freedom for user-specific installations.

**Comment by kyoforkshomebrews** [comment]
+1, My root drive is filling up...

**Comment by 17314642** [comment]
+1, i sometimes move flatpak apps and data between distros, and I always backup /home, but always forget about system flatpak directory

**Comment by VictorVoltzz** [comment]
+1, while this function is not avaiable, i recommend installing the Easy Flatpak store app and setting it to user mode on configs

**Comment by ghost** [comment]
Not really familiar with flatpaks, but wouldn't this step remove the out of the box permissions? I remember there were times, when I needed flatseal to lift permissions for flatpaks to behave as they should, but nowadays, flatpaks just works with the right permissions, without me ever needing to manually intervene into the permissions. Isn't this --system assuring the correct permissions? Wouldn't the --user limit these perms, and again would need the user to manually set the perms?

**Reply by 17314642** [reply]
no, --system and --user is just flatpak data location, sandbox is always launched the same way
